src/packages/cli/src/args.ts

```diff
@@ -33,7 +33,13 @@ function unescapeEntities(html: string) {
     .replace(/&#39;/g, "'");
 }
 const highlight = (t: string) => unescapeEntities(marked.parseInline(t));
-const center = (t: string) => " ".repeat((wrapWidth - t.length) >> 1) + t;
+const center = (str: string) =>
+  wrapWidth ? " ".repeat((wrapWidth - str.length) >> 1) + str : str;
```

Comment:
This won't work if `str.length > wrapWidth`. This probably should be

```suggestion
const center = (str: string) => " ".repeat(Math.max(0, (wrapWidth - str.length) >> 1)) + str;
```

Reply:
good catch. fixed in [`17196cd` (#721)](https://github.com/trufflesuite/ganache-core/pull/721/commits/17196cdab81f971476dcdbaa84bee79975b7309a)

got rid of the bitshift math since this isn't a 1 liner anymore